operator/controllers/syncer/csi_syncer.go

```diff
@@ -527,7 +527,10 @@ func (s *csiControllerSyncer) ensureAttacherContainersSpec() []corev1.Container
 	attacher := s.ensureContainer(attacherContainerName,
 		s.getSidecarImage(config.CSIAttacher),
 		// TODO: make timeout configurable
-		[]string{"--v=5", "--csi-address=$(ADDRESS)", "--resync=10m", "--timeout=2m", "--leader-election=true", "--http-endpoint=:" + fmt.Sprint(config.LeaderLivenessPort)},
+		[]string{"--v=5", "--csi-address=$(ADDRESS)", "--resync=10m", "--timeout=2m",
+			"--leader-election=true", "--leader-election-lease-duration=137s",
```

Comment:
can we have common variable instead on setting value explicitly
something like 
            - --leader-election-lease-duration=${LEADER_ELECTION_LEASE_DURATION}
            - --leader-election-renew-deadline=${LEADER_ELECTION_RENEW_DEADLINE}
            - --leader-election-retry-period=${LEADER_ELECTION_RETRY_PERIOD}
            
           This will help change values from single location